testsuite/cluster-test/src/tx_emitter.rs

```diff
@@ -67,12 +71,50 @@ struct StatsAccumulator {
     latency: AtomicU64,
 }
 
+pub struct AtomicHistogram {
+    buckets: Box<[AtomicU64]>,
```

Comment:
I think you should be able to just use `Vec<AtomicU64>` instead of `Box<[AtomicU64]>` here.